cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
+===================================
+
+.. note:: It is recommended to read `Extend admin Ui`_ beforehand to get a better understandig of how Sulu admin classes work.
```

Comment:
The link is not working, and could you please change the wording to "Extend Admin UI", to match the case int he other page?

Reply:
Not sure why the link is not working. Can you have a look?